tests/cases/fourslash/refactorConvertToOptionalChainExpressionForTriggerReason1.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path='fourslash.ts' />
+
+////let a = { b: { c: 0 } };
+////a && a.b && /*a*//*b*/a.b.c;
+
+// Only offer refactor for empty span if explicity requested
+goTo.select("a", "b");
+verify.not.refactorAvailableForTriggerReason("implicit", "Convert to optional chain expression");

```

Comment:
`"invoked"` is sent when an editor performs a request refactor command (for example by pressing `ctrl` + `shift` + `r` in VSCode). In most cases we use this when a refactor is requested for an empty span so we know to walk up the tree to find an appropriate node for the refactor rather than using a selected span (since there isn't one).

In the future we may also want to use this to more aggressively search for refactors around a span when refactors are explicitly requested as opposed to a request triggered by simply selecting the span.